src/main/java/net/bramp/ffmpeg/info/Layout.java

```diff
@@ -0,0 +1,5 @@
+package net.bramp.ffmpeg.info;
+
+public interface Layout {
```

Comment:
Should this be called ChannelLayout?

Reply:
I used Layout because it's the name used by ffmpeg. But I agree, your suggestion is better.